packages/react-components/src/theming/icons.tsx

```diff
@@ -74,6 +76,10 @@ export const DEFAULT_COMPONENT_ICONS = {
   ControlButtonParticipants: <People20Filled />,
   ControlButtonScreenShareStart: <ShareScreenStart20Filled />,
   ControlButtonScreenShareStop: <ShareScreenStop20Filled />,
+  /* @conditional-compile-remove(file-sharing) */
+  Cancel: <FontIcon iconName="Cancel" />,
```

Comment:
Can we use icons from `'@fluentui/react-icons'` here to move towards having consistently bundled icons and not pull from fluent's CDN (and avoid this: [Bug 2821183](https://skype.visualstudio.com/SPOOL/_workitems/edit/2821183): [Web] Flaky tests: File sharing download icon)

Reply:
(not blocking for this PR, just suggestion)

Reply:
Absolutely. I overlooked that. Thanks for the headsup

Reply:
Fixed